frontend/apps/reader/modules/readerfont.lua

```diff
@@ -209,11 +209,17 @@ end
 
 function ReaderFont:makeDefault(face)
     if face then
-        UIManager:show(ConfirmBox:new{
-            text = T( _("Set default font to %1?"), face),
-            ok_callback = function()
+        UIManager:show(MultiConfirmBox:new{
+            text = T( _([[Set %1 as default or fallback font? The fallback font ]] 
```

Comment:
In my opinion that'd be neither ugly nor bad style, rather the opposite. But I didn't write the style guide. ;) Anyway, it wasn't an issue [here](https://github.com/koreader/koreader/blob/954c2c941613f41120e8e356aba1f46a9fabbd05/frontend/apps/filemanager/filemanagersetdefaults.lua#L51).

The string concatenation thing just bugs me, even though there's a very simple and obvious fix: don't do it. xD